src/Microsoft.ML.AutoML/TrainerExtensions/SweepableParams.cs

```diff
@@ -115,6 +117,15 @@ public static IEnumerable<SweepableParam> BuildLightGbmParams()
             };
         }
 
+        public static IEnumerable<SweepableParam> BuildMatrixFactorizationParams()
+        {
+            return new SweepableParam[]
+            {
+                new SweepableDiscreteParam(nameof(MatrixFactorizationTrainer.Options.NumberOfIterations), new object[] { 10, 20, 40 }),
+                new SweepableFloatParam(nameof(MatrixFactorizationTrainer.Options.LearningRate), 0.001f, 0.1f, numSteps: 2),
```

Comment:
How come this is different than:

[https://github.com/dotnet/machinelearning/blob/461b7d38c06d64e53c76e29770665c4b0bb44e72/src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs#L216-L218](https://github.com/dotnet/machinelearning/blob/461b7d38c06d64e53c76e29770665c4b0bb44e72/src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs#L216-L218)

Reply:
OK changing 
```
new SweepableFloatParam(nameof(MatrixFactorizationTrainer.Options.LearningRate), 0.001f, 0.1f, numSteps: 2),
```
to 
```
new SweepableDiscreteParam(nameof(MatrixFactorizationTrainer.Options.LearningRate), new object[] { 0.001f, 0.01f, 0.1f }),
```
in next commit